mobile-app/app/screens/AppNavigator/screens/Loans/screens/PaybackLoanScreen.tsx

```diff
@@ -196,6 +197,10 @@ export function PaybackLoanScreen ({
       .catch(logger.error)
   }, [])
 
+  useEffect(() => {
+    return setIsValidInput(new BigNumber(amountToPay).lte(0) || amountToPay === '')
```

Comment:
Is your naming correct? Valid input should be lte 0?

If yes, maybe we can change amountToPay === '' to isNaN